android/app/gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
what is this file

Reply:
I don't know. it was generated automatically. I will try to remove and rebuild